network-api/networkapi/wagtailpages/static/js/directory-listing-filters.js

```diff
@@ -130,7 +130,7 @@
    * associated year (for now)
    */
   function bindEventsToLabels() {
-    let labels = Array.from(filters);
+    const labels = filters;
```

Comment:
Yep that makes sense. I could also move all of this from the "global" scope
https://github.com/mozilla/foundation.mozilla.org/blob/42e813f9d9851a0c72677f23081d0843eb443b24/network-api/networkapi/wagtailpages/static/js/directory-listing-filters.js#L5-L7
inside `bindEventsToLabels()` since it's not used anywhere else.